eth/api.go

```diff
@@ -1168,7 +1168,7 @@ func (tx *Tx) UnmarshalJSON(b []byte) (err error) {
 		tx.GasLimit = rpc.NewHexNumber(0)
 	}
 	if tx.GasPrice == nil {
-		tx.GasPrice = rpc.NewHexNumber(int64(50000000000))
+		tx.GasPrice = rpc.NewHexNumber(0)
```

Comment:
That is why I explicitly asked for feedback. I've no idea what this is used for, but the value 50 shannon is definitely wrong there either way.